src/query/graphite/native/builtin_functions_test.go

```diff
@@ -3193,6 +3193,32 @@ func TestConsolidateBy(t *testing.T) {
 	require.NotNil(t, err)
 }
 
+func TestPow(t *testing.T) {
+	var (
+		ctx           = common.NewTestContext()
+		millisPerStep = 10000
+		output        = []float64{1.0, 4.0, 9.0, 16.0, 25.0}
+	)
+
+	defer func() { _ = ctx.Close() }()
+
+	series := ts.NewSeries(
+		ctx,
+		"foo",
+		ctx.StartTime,
+		common.NewTestSeriesValues(ctx, millisPerStep, []float64{1.0, 2.0, 3.0, 4.0, 5.0}),
+	)
+	results, err := pow(ctx, singlePathSpec{
+		Values: []*ts.Series{series},
+	}, 2)
```

Comment:
Can you also add a test that has multiple time series as the input?

Reply:
I've added the test, and its passing all the checks now